docs/ros/config/yaml.mdx

```diff
@@ -6,3 +6,340 @@ toc_min_heading_level: 2
 toc_max_heading_level: 4
 ---
 
+Our robots provide users with a wide range of customization options: sensors, sensor mounting structures, and custom-made parts. Matching the flexibility of our hardware, Clearpath's ROS2 system is designed to keep all these customization decisions in a single configuration file.
+
+The **Clearpath Robot Configuration YAML**, or `robot.yaml` for short, contains all information pertinent to the entire robot system, allowing robot builders and users to quickly and easily modify any ROS2 component.
+
+## Overview
+
+The `robot.yaml` is composed of five major sections:
+1. [**system**](#system) level information such as the robot's hostname, IP, and ROS middleware implementation.
+2. [**platform**](#platform) level configurations such as robot specific mounting structures and specifying an extra URDF to attach.
+3. [**accessories**](#accessories) that are URDF primitives: boxes, cylinders, and meshes.
+4. [**mounts**](#mounts) that are predefined, generic, sensor mounting structures.
+5. [**sensors**](#sensors) that are an inventory of Clearpath supported sensors.
+
+Additionally, there are two other, required parameters:
+1. The robot's [**serial_number**](#serial-number); which is used to determine the model and version.
+2. The YAML **version** to use, as future updates are released.
+
+#### Sample
+Below is a sample **J100 Jackal** robot YAML. In the following sections, we will reference each and every component of this sample file.
```

Comment:
Should this be 'Jackal J100' similar to Husky's naming.

Reply:
Done